### DISCUSSION TITLE
GitHub Pages doesnt publish anymore

### DISCUSSION THREAD

**Discussion by 4chi11e** [community/community#23589] [Other Feature Feedback, Questions, & Ideas]
<p>I used github pages for my jekyll website for a while and it always worked well. Today I noticed that it stopped publishing anything and I dont know why (you can see the screen of the page below). Im no expert and you can see my english is quite bad. I don’t know what i can do. Can you help me?</p>
<p><a href="https://4chi11e.github.io/manuale-libero-online-di-informatica/" class="onebox" target="_blank" rel="noopener nofollow ugc">https://4chi11e.github.io/manuale-libero-online-di-informatica/</a></p><aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/4chi11e/manuale-libero-online-di-informatica/">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/4chi11e/manuale-libero-online-di-informatica/" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/16581760/181078084-726184c1-20b0-444d-9e9a-95d0e4d0ef33.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/6/367a0fed8178e569313b03f0bf272c1479a91e4e_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/6/367a0fed8178e569313b03f0bf272c1479a91e4e_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/6/367a0fed8178e569313b03f0bf272c1479a91e4e.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/6/367a0fed8178e569313b03f0bf272c1479a91e4e_2_10x10.png"></div>

<h3><a href="https://github.com/4chi11e/manuale-libero-online-di-informatica/" target="_blank" rel="noopener nofollow ugc">GitHub - 4chi11e/manuale-libero-online-di-informatica: Sito-Dispensa di...</a></h3>

  <p>Sito-Dispensa di Informatica per il Liceo Scientifico delle Scienze Applicate - GitHub - 4chi11e/manuale-libero-online-di-informatica: Sito-Dispensa di Informatica per il Liceo Scientifico delle Sc...</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/f/8fd3a017cfda80cb21f5763815e2c96b5b979858.png" data-download-href="/uploads/short-url/kwlGnsVCKiit897KJALgojpFv3W.png?dl=1" title="screen-github-pages" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/16581760/181078086-82651eeb-129e-4c9c-b5f3-1c1b3f9c50cd.png" alt="screen-github-pages" data-base62-sha1="kwlGnsVCKiit897KJALgojpFv3W" width="690" height="476" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/f/8fd3a017cfda80cb21f5763815e2c96b5b979858_2_690x476.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/f/8fd3a017cfda80cb21f5763815e2c96b5b979858_2_1035x714.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/8/f/8fd3a017cfda80cb21f5763815e2c96b5b979858.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/f/8fd3a017cfda80cb21f5763815e2c96b5b979858_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">screen-github-pages</span><span class="informations">1133×783 42.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment]
<p>you have several build failures,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/0/40be9f82f934a48a71f3f8a38ef72e28f5a62a3b.png" data-download-href="/uploads/short-url/9eKXU4AyRGKEe39Ggli7Xk8t9Mv.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181078087-380f17a4-f666-4323-b7c3-e41c75f6f85b.png" alt="image" data-base62-sha1="9eKXU4AyRGKEe39Ggli7Xk8t9Mv" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/0/40be9f82f934a48a71f3f8a38ef72e28f5a62a3b_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/0/40be9f82f934a48a71f3f8a38ef72e28f5a62a3b_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/0/40be9f82f934a48a71f3f8a38ef72e28f5a62a3b_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/0/40be9f82f934a48a71f3f8a38ef72e28f5a62a3b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 94.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>if you are using an external service that might depend on submodules, that can be the problem,</p>
<p>I’m not yet checking all the details of your site,</p>

**Comment by 4chi11e** [comment]
<p>I’m so ignorant that I didn’t even know about the page you showed me. I looked the details of one of the deployment errors and it seems the problem is that it cannot find the theme (just-the-docs <a href="https://pmarsceill.github.io/just-the-docs/" class="inline-onebox" rel="noopener nofollow ugc">Home | Just the Docs</a>), but I didnt change any configuration and in the past it always worked. Locally on my PC continue to work normally. I dont understand what i have to do.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/e/e/ee54c8f66fd3a62c142452df1a03b7f80673b3fd.png" data-download-href="/uploads/short-url/y0nhUY446bjgsHko9YhGcHuz1Dv.png?dl=1" title="deploy-error" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/16581760/181078088-adc9e62d-9e27-4e4a-8117-b4a7be1fec6e.png" alt="deploy-error" data-base62-sha1="y0nhUY446bjgsHko9YhGcHuz1Dv" width="690" height="344" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/e/ee54c8f66fd3a62c142452df1a03b7f80673b3fd_2_690x344.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/e/ee54c8f66fd3a62c142452df1a03b7f80673b3fd_2_1035x516.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/e/e/ee54c8f66fd3a62c142452df1a03b7f80673b3fd_2_1380x688.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/e/ee54c8f66fd3a62c142452df1a03b7f80673b3fd_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">deploy-error</span><span class="informations">2501×1247 206 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by airtower-luna** [comment, marked as answer]
<p>I read there were some changes to Pages recently to make them work through Actions (which gives you access to those logs, so that’s actually new). The documentation says you can use one of a predefined set of themes, or one hosted on GitHub (step 5): <a href="https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/adding-a-theme-to-your-github-pages-site-using-jekyll" class="inline-onebox">Adding a theme to your GitHub Pages site using Jekyll - GitHub Docs</a></p>

**Comment by 4chi11e** [comment]
<p>It worked!!!<br>
In my _config.yml file i had to change “theme: just-the-docs” to “remote_theme: pmarsceill/just-the-docs” as suggested in the page you linked me.<br>
Thank you very much!</p>
